deps/wazuh_testing/wazuh_testing/tools/agent_simulator.py

```diff
@@ -608,37 +672,159 @@ def set_module_attribute(self, module_name, attribute, value):
+            event_data['name'] = f"CIS Benchmark for {self.os}"
+            event_data['policy_id'] = f"cis_{self.os}_linux"
+            event_data['file'] = f"cis_{self.os}_linux.yml"
+            event_data['description'] = 'This provides prescriptive guidance for establishing a secure configuration.'
+            event_data['references'] = 'https://www.cisecurity.org/cis-benchmarks'
+            total_checks = randint(0, 900)
+            passed_checks = randint(0, total_checks)
+            failed_checks = randint(0, total_checks - passed_checks)
+            invalid_checks = total_checks - failed_checks - passed_checks
+            event_data['passed'] = passed_checks
+            event_data['failed'] = failed_checks
+            event_data['invalid'] = invalid_checks
+            event_data['total_checks'] = total_checks
+            event_data['score'] = 20
+            event_data['start_time'] = self.started_time
+            self.started_time = int(time() + 1)
+            event_data['end_time'] = self.started_time
+            event_data['hash'] = getrandbits(256)
+            event_data['hash_file'] = getrandbits(256)
+            event_data['force_alert'] = '1'
+            return event_data
+
+        def create_check_sca_event(event_data):
+            event_data['type'] = 'check'
+            event_data['id'] = randint(0, 9999999999)
+            event_data['policy'] = f"CIS Benchmark for {self.os}"
+            event_data['policy_id'] = f"cis_{self.os}_policy"
+            event_data['check'] = {}
+            event_data['check']['id'] = randint(0, 99999)
+            event_data['check']['title'] = 'Ensure root is the only UID 0 account'
+            event_data['check']['description'] = 'Any account with UID 0 has superuser privileges on the system'
+            event_data['check']['rationale'] = 'This access must be limited to only the default root account'
+            event_data['check']['remediation'] = 'Remove any users other than root with UID 0'
+            event_data['check']['compliance'] = {}
+            event_data['check']['compliance']['cis'] = '6.2.6'
+            event_data['check']['compliance']['cis_csc'] = '5.1'
+            event_data['check']['compliance']['pci_dss'] = '10.2.5'
+            event_data['check']['compliance']['hipaa'] = '164.312.b'
+            event_data['check']['compliance']['nist_800_53'] = 'AU.14,AC.7'
+            event_data['check']['compliance']['gpg_13'] = '7.8'
+            event_data['check']['compliance']['gdpr_IV'] = '35.7,32.2'
+            event_data['check']['compliance']['tsc'] = 'CC6.1,CC6.8,CC7.2,CC7.3,CC7.4'
+            event_data['check']['rules'] = 'f:/etc/passwd -> !r:^# && !r:^\\\\s*\\\\t*root: && r:^\\\\w+:\\\\w+:0:\"]'
+            event_data['check']['condition'] = 'none'
+            event_data['check']['file'] = '/etc/passwd'
+            event_data['check']['result'] = choice(['passed', 'failed'])
+            return event_data
```

Comment:
```suggestion

            return event_data
```

Reply:
This neither.

Reply:
This was also solved 5 days ago https://github.com/wazuh/wazuh-qa/commit/6976fb26ae1e4fcd0d905a309eb1820db1111a5c by @Rebits